spec/models/manageiq/providers/redhat/infra_manager/provision/state_machine_spec.rb

```diff
@@ -70,6 +70,8 @@ def test_poll_clone_complete
     @test_poll_clone_complete_setup ||= begin
       expect(task).to receive(:clone_complete?).and_return(false, false, true)
       expect(task).to receive(:requeue_phase).twice { requeue_phase }
+      # make sure that full refresh is not run
+      expect(EmsRefresh).not_to receive(:queue_refresh).with(no_args)
```

Comment:
I thought it as well but there is a targeted refresh when stopping. If I remove it the tests fails.